### ISSUE TITLE
Remove old RC, Alpha and Beta Downloads?

### ISSUE THREAD

**Issue by rprouse** [nunit/nunit.github.io#11]
The old downloads lists are very long. I'm not sure how useful old RC, alpha and beta releases are to people. Should we remove them from the lists?

We could also put the previous releases in a [collapsible group](http://getbootstrap.com/javascript/#collapse) to clean up the page.

**Comment by CharliePoole** [comment]
IIRC there was an issue to put the legacy stuff on a separate page for both downloads and docs. Another issue spoke to converting the site structure to be product-based, i.e. products at the top level with downloads, docs, etc. beneath that.

I can go through the nunit.org issues and move any that seem appropriate.

**Comment by rprouse** [comment]
@CharliePoole I am sure you are really busy right now. I don't mind going through the NUnit.org issues and moving them.

**Comment by CharliePoole** [comment]
I'd like a chance to express an opinion on them.

**Comment by CharliePoole** [comment]
@rprouse I hadn't expected any of this stuff to start quite yet - not until I finished with my tasks - but I'm doing my best to keep up with you!

**Comment by ChrisMaddock** [comment]
Collapsible groups sound good to me. I'd prefer that to removing them entirely - they are still releases, which are out in the wild. 🙂

**Comment by rprouse** [comment]
No worries about the RCs, it is just an idea. We can clean them up in other ways 😄
